core-metastore/src/main/java/org/zalando/nakadi/repository/db/EventTypeRepository.java

```diff
@@ -102,6 +103,18 @@ public EventType mapRow(final ResultSet rs, final int rowNum) throws SQLExceptio
                 new EventTypeMapper());
     }
 
+    public List<EventType> list(final String[] writers){
+        return jdbcTemplate.query(
+                String.format("SELECT et_event_type_object\n" +
+                        "FROM zn_data.event_type," +
+                        "jsonb_to_recordset(et_event_type_object->'authorization'->'writers')" +
+                        "as writers(value text)\n" +
```

Comment:
I also think we should match both `data_type` and `value`.  The query string can look like `?writer=user:jdoe&writer=service:appname` — this is consistent with the error reports on wrong authorization attributes at least.

Reply:
This would impact the convenience on both ends. Users will have to follow a format that we have to describe and we'd need to do a bit of parsing. I was considering get with body, but post is already taken off.

Reply:
But without specifying user or service it is ambiguous and we can avoid that by using `type:value` format: that's not too much parsing for being future-proof, IMO.